projects/demo/src/modules/components/slider-old/examples/import/insert-template.txt

```diff
@@ -0,0 +1,6 @@
+<tui-slider [formControl]="control"
```

Comment:
what do you think about using markdown? as it is in the example with the avatar demo